src/_pages-content/health-and-safety.md

```diff
@@ -0,0 +1,44 @@
+
+### MASKS ARE REQUIRED
+We are asking you to wear a mask during your time at PyCon India. That includes talks, the expo hall, hallways and sprints. We are looking into providing a second eating area outdoors for attendees who would like that – for either the airflow or the vitamin D or both.
+
+### Exceptions are:
+* Outdoor spaces
+* Indoors while actively consuming food or beverages
+* Speakers while presenting
+* While necessary for communicating with someone who is hearing impaired when the ability to see the mouth is essential for communication
+* Very briefly to take photos
+
+Masks must be worn over the nose and mouth and must be made of a tight-knit, non-permeable material. N95 or equivalent masks such as KN95 or K95 are recommended. Alternatives to face coverings such as face shields, loose gaiters, scarves, bandanas, and face coverings with ventilation are not permitted to be used as masks. Although we encourage you to bring your own masks that you find comfortable, there will also be free masks available for attendees during each day of PyCon India, as supplies permit. Want to be our Mask Sponsor? [Get in touch!](https://mail.python.org/mailman3/lists/inpycon-announce.python.org/)
+
+Attendees who are repeatedly unwilling to comply with the mask requirements will be asked to leave and are not eligible for a refund. Please don’t be “this person.”
+
+
+### VACCINATIONS ARE ENCOURAGED
+We encourage PyCon India attendees to stay up to date with the vaccinations that are available to them where they live. We know that PyCon India attendees are coming from all over the world and the availability of current vaccines is extremely different from place to place. We encourage you to do your best.
+
+This does represent a change from the most recent two PyCon India events; a verified vaccination status was required to attend both PyCon India 2022 and 2023. We have made the decision not to require vaccination for a few reasons, including the increasingly wide variation in availability and recommendations for vaccinations and boosters globally, changes over time in the relevance of vaccination status in risk calculation, as well as the significant cost of the verification procedure which we can redirect to other safety measures.
+
+Like the rest of this policy, we may review this stance and consider adding additional precautions should the situation change (see final paragraph of this policy).
+
+### RAPID COVID-19 TESTS ARE ENCOURAGED
+We encourage PyCon India attendees to use at-home/antigen/rapid tests before they arrive and while they are at PyCon India. We will be providing free additional rapid tests as supplies permit to anyone who needs them in the staff office.
+
+### ATTEND ONLINE
```

Comment:
We don't have online.

Reply:
i have to remove that?

Reply:
Yes of course. We don't have online. Did we mention about online?